r/R/table.R

```diff
@@ -254,6 +257,49 @@ names.Table <- function(x) x$ColumnNames()
 #' @export
 `[[.Table` <- `[[.RecordBatch`
 
+#' @export
+`[[<-.Table` <- function(x, i, value) {
+  if (is.null(value)) {
+    if (is.character(i)) {
+      i <- match(i, names(x)) - 1L
+    }
+    x <- x$RemoveColumn(i)
+  } else {
+    if (!is.character(i)) {
+      # get or create a/the column name
+      if (i <= ncol(x)) {
```

Comment:
This is a trivial difference but it does save some work because `ncol == function (x) dim(x)[2L]`

```suggestion
      if (i <= x$num_columns) {
```

Reply:
I went looking for a function call to `num_columns()` and should have looked for an active binding, oops!